great_expectations/rule_based_profiler/parameter_builder/simple_dateformat_string_parameter_builder.py

```diff
@@ -99,20 +98,23 @@ def _build_parameters(
             parameters=parameters,
         )
 
-        # TODO: <Alex>ALEX -- Is the check below still relevant?</Alex>
         if len(batch_ids) > 1:
             logger.warning(
-                f"Parameter Builder {self.name} received {len(batch_ids)} batches but can only process one."
+                f"Parameter Builder {self.name} received {len(batch_ids)} batches but will utilize only the first one."
```

Comment:
Considering that when we load a Validator with multiple batches, the ActiveBatch winds up being the last one loaded, I am wondering if there is an argument to utilize the last batch rather than the first in this case.

Reply:
@talagluck Any batch is OK.  But this came from the prototype, which is here for example purposes only (for now); so I prefer that we change anything later when we actually utilize this parameter builder.  Thanks.